impl/src/main/java/org/opendaylight/unimgr/mef/nrp/impl/topologytervice/TapiTopologyServiceImpl.java

```diff
@@ -13,6 +13,8 @@
 import org.opendaylight.controller.md.sal.common.api.data.LogicalDatastoreType;
 import org.opendaylight.controller.md.sal.common.api.data.ReadFailedException;
 import org.opendaylight.unimgr.mef.nrp.common.NrpDao;
+import org.opendaylight.yang.gen.v1.urn.odl.unimgr.yang.unimgr.ext.rev170531.Node1;
+import org.opendaylight.yang.gen.v1.urn.odl.unimgr.yang.unimgr.ext.rev170531.Node2;
```

Comment:
maybe we could start naming the augmentations unimgr-ext is our file so we can do that 

to do that you need to import yang-ext and use ext:augment-identifier "xxx" then instead Node1... you will have XXX